GeoLib/Grid.h

```diff
@@ -506,8 +502,8 @@ void Grid<POINT>::initNumberOfSteps(std::size_t n_per_cell,
 
 	switch (dim.count()) {
 	case 3: // 3d case
-		_n_steps[0] = sc_ceil(std::cbrt(
-			n_pnts*delta[0]*delta[0]/(n_per_cell*delta[1]*delta[2])));
+		_n_steps[0] = sc_ceil(std::cbrt(n_pnts * (delta[0] / delta[1]) *
+			                            (delta[0] / delta[2]) / n_per_cell));
```

Comment:
mixed tabs + whitespaces.
also, could you please explain why the computation has been changed (and why it has only been changed for n_steps[0])